packages/panels/src/components/notifications/NotificationHeader.tsx

```diff
@@ -36,13 +38,18 @@ export const NotificationHeader: React.FC<NotificationHeaderProps> = ({
   onClose,
   unread = false,
   variant,
+  iconBackgroundColor,
 }) => {
   const circledIcon = icon ? (
     <CircledIcon
       size={"small"}
       icon={icon}
       aria-label={iconAriaLabel}
-      backgroundColor={getIconBgColor(unread, variant)}
+      backgroundColor={
```

Comment:
There is a slight chance that iconBackgroundColor will be removed, and be replaced by variants.

So `<IconColor variant={"success"} ..>`, instead of developers needing to know what color success is.

But you don't have to care about that in this PR, but I thought it would be good to know. When that happens, we can just have derive IconColor variant from Notification variant.